### ISSUE TITLE
What does AllMasterSlavesStale mean and why this error occur?

### ISSUE THREAD

**Issue by umnya** [openark/orchestrator#806]
Hello, I am a new in orchestrator.
I set up 3 mysql server(1 master and 2 slaves).
Replication uses GTID, and Mysql version is 5.7.
And I added these three servers in orchestraotor.
But this cluster often has "AllMasterSlavesStale" state.
I didn't change anything, and db's health looks good.
So I wonder why this message is occurred and what I should look into.

**Comment by jackiesre721** [comment]
I think this is normal, because the master-slave architecture has not changed.

**Comment by shlomi-noach** [comment]
@liuqian1990 `AllMasterSlavesStale` has been deprecated perhaps one year ago. Can you please clarify which `orchestrator` version you are using? Please make sure to grab the [latest](https://github.com/github/orchestrator/releases/latest)

**Comment by umnya** [comment]
Hi, thank you for your answer.
I reinstalled latest version.
But I got some strange situation when I executed discover menu.
I got failed to discover new server when I use web UI.
But I was successful when I use cli.
I could see cluster info and topology, but I couldn't see this information in web UI.
In cli
$ ./orchestrator -c topology -i 10.xxx.xxx.xxx:3306 cli
10.xxx.xxx.xxx:3306                              [0s,ok,5.7.24-27-log,rw,ROW,>>,GTID]
+ 10.xxx.xxx.xxx.hostname:3306 [0s,ok,5.7.24-27-log,ro,ROW,>>,GTID]
+ 10.xxx.xxx.xxx.hostname:3306 [0s,ok,5.7.24-27-log,ro,ROW,>>,GTID]

In Web UI
![image](https://user-images.githubusercontent.com/8628614/52933442-b23d8d00-3396-11e9-8ce3-950c42dfac92.png)
